metaflow/plugins/cards/card_decorator.py

```diff
@@ -0,0 +1,192 @@
+import subprocess
+import os
+import sys
+import json
+import tempfile
+from metaflow.decorators import StepDecorator, flow_decorators
+from metaflow.current import current
+from metaflow.util import to_unicode
+
+# from metaflow import get_metadata
+import re
+
+CARD_ID_PATTERN = re.compile(
+    "^[a-zA-Z0-9_]+$",
+)
+
+
+class CardDecorator(StepDecorator):
+    name = "card"
+    defaults = {
+        "type": None,
+        "options": {},
+        "scope": "task",
+        "timeout": 45,
+        "save_errors": True,
+    }
+
+    def __init__(self, *args, **kwargs):
+        super(CardDecorator, self).__init__(*args, **kwargs)
+        self._task_datastore = None
+        self._environment = None
+        self._metadata = None
+        # todo : first allow multiple decorators with a step
+
+    def add_to_package(self):
+        return list(self._load_card_package())
+
+    def _load_card_package(self):
+        try:
+            import metaflow_cards
```

Comment:
do we need this for now? If not, I would strip so we can come up with a unified story about adding cards/extensions and what not. I think there can be a good unified story here.